src/sanity.c

```diff
@@ -1118,7 +1118,7 @@ rand_password(void)
 static void
 create_lostandfound(dbref * player, dbref * room)
 {
-    char *player_name = "lost+found";
+    char player_name[tp_player_name_limit+1];
```

Comment:
This is probably what Windows is complaining about.  Most compilers can't do dynamic size allocations of heap variables like this, I don't think.  You have two options -- use a static size, or do like:

```
char * player_name = (char*) malloc(tp_player_name_limit+1);
```
and don't forget to free on returns.  I think it is probably important to address this as it could break in random other compilers.